hands-on/launch-app.html.md.erb

```diff
@@ -10,55 +10,65 @@ redirect_from:
 
 ---
 
-Fly.io allows you to deploy any kind of app as long as it is packaged in a **Docker image**. That also means you can just deploy a Docker image and as it happens we have one ready to go in `flyio/hellofly:latest`. 
+Fly.io enables you to deploy almost any kind of app using a Docker image. To learn more about the different ways to get your app ready to deploy, refer to [Launch a New App on Fly.io](/docs/apps/launch/).
 
-Each Fly.io application needs a `fly.toml` file to tell the system how we'd like to deploy it. That file can be automatically generated with the `flyctl launch` command, which will ask a few questions to set everything up. Let's run through it now: 
+For this example, you can use our pre-built Docker image, `flyio/hellofly:latest`, to try out creating and deploying an app. 
+
+Each Fly.io application needs a `fly.toml` file to tell the system how to deploy it. The `fly.toml` file can be automatically generated with the `flyctl launch` command, which will ask a few questions to set everything up. Let's run through it now:
 
 ```cmd
 fly launch --image flyio/hellofly:latest
 ```
 ```output 
-? App Name (leave blank to use an auto-generated name):
+? Choose an app name (leave blank to generate one):
 ```
-Here you can enter the name of the app. Please note that you can only use numbers, lowercase letters and dashes.
+
+Here you can enter the name of the app. App names can include only numbers, lowercase letters, and dashes.
+
 ```output 
 ? Select organization: Personal (personal)
 ```
-**Organizations**: Organizations are a way of sharing applications and resources between Fly.io users. Every Fly.io account has a personal organization, called `personal`, which is only visible to your account. Let's select that for this guide.
+
+**Organizations**: Organizations are a way of sharing applications and resources between Fly.io users. Every Fly.io account has a personal organization, called `personal`, which is only visible to your account. Select the 'personal' organization for this example.
 
 <div class="callout">
-    If you did not add another organization to your account, the `personal` organization will be selected automatically.  
+    If you didn't add another organization to your account, the `personal` organization is selected automatically.  
 </div>
 
 Next, you'll be prompted to select a region to deploy in.
 
 ```output
-? Select region: ord (Chicago, Illinois (US))
+? Choose a region for deployment: Chicago, Illinois (US) (ord)
 ```
 
-At this point, flyctl creates a shell of an app for you and writes a starter configuration to a `fly.toml` file. 
+At this point, `flyctl` creates a shell of an app for you and writes a starter configuration to a `fly.toml` file. 
```

Comment:
"flyctl" is the name of the tool. I don't think we want to change it to pre formatting everywhere. We might like to capitalize it. I've never been inclined to, but we can make a decision (Discourse) and put it in the style reference.